src/app/public/modules/inline-delete/fixtures/inline-delete.component.fixture.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Component,
+  ViewChild
+} from '@angular/core';
+
+import {
+  SkyInlineDeleteComponent
+} from '../inline-delete.component';
+
+@Component({
+  selector: 'sky-test-cmp',
+  templateUrl: './inline-delete.component.fixture.html',
+  styles: [`
+    #inline-delete-fixture {
+      height: 400px;
+      position: relative;
+      width: 400px;
+    }
+
+    #hidden-button {
+      visibility: hidden;
+    }
+  `]
+})
+export class InlineDeleteTestComponent {
+
+  @ViewChild(SkyInlineDeleteComponent)
+  public inlineDelete: SkyInlineDeleteComponent;
```

Comment:
Your `ViewChild` should be last in order.

Reply:
Done